calibration/FLORTD/CGINS-FLORTD-01102__20220105.csv

```diff
@@ -0,0 +1,11 @@
+serial,name,value,notes
+1102,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1102_Calibration_Files_2022-01-05.zip > BBFL2W-1102.dev
+1102,CC_dark_counts_cdom,49,
+1102,CC_dark_counts_chlorophyll_a,53,
+1102,CC_dark_counts_volume_scatter,75,
+1102,CC_depolarization_ratio,0.039,Constant
+1102,CC_measurement_wavelength,700,Constant
+1102,CC_scale_factor_cdom,9.070E-02,
+1102,CC_scale_factor_chlorophyll_a,1.180E-02,
+1102,CC_scale_factor_volume_scatter,2.242E-06,
+1102,CC_scattering_angle,124,Constant
```

Comment:
OK, I figured out that there is a post deployment calibration for all three channels, and that's where the scattering coeffs and the 1/5 date come from since there was no subsequent characterization done on that channel. So this csv is good, but the hard copies should probably be corrected by the vendor.

Reply:
adding comment for training demo